src/Service/ProcessServiceImpl.cpp

```diff
@@ -62,7 +63,7 @@ Status ProcessServiceImpl::GetProcessList(ServerContext*, const GetProcessListRe
 Status ProcessServiceImpl::GetModuleList(ServerContext* /*context*/,
                                          const GetModuleListRequest* request,
                                          GetModuleListResponse* response) {
-  int32_t pid = request->process_id();
+  uint32_t pid = request->process_id();
```

Comment:
this should probably be int32_t pid = ToNative...

Reply:
done.